src/index.js

```diff
@@ -257,6 +308,7 @@ const loadDataOnPage = () => {
       ddb.totals = newTotals[0];
       ddb.totalsDiff = newTotals[1];
       ddb.trend = summaryData.daily;
+      console.log(ddb.trend);
```

Comment:
Maybe we can get rid of this when we deploy?

Reply:
Good catch, fixing.